core/chain_makers.go

```diff
@@ -102,7 +102,7 @@ func (b *BlockGen) SetParentBeaconRoot(root common.Hash) {
 		blockContext = NewEVMBlockContext(b.header, b.cm, &b.header.Coinbase)
 		vmenv        = vm.NewEVM(blockContext, vm.TxContext{}, b.statedb, b.cm.config, vm.Config{})
 	)
-	ProcessBeaconBlockRoot(root, vmenv, b.statedb)
+	ProcessBeaconBlockRoot(root, vmenv, b.statedb, b.Number(), b.Timestamp())
```

Comment:
Please replace it with `github.com/ethereum/go-verkle`

Reply:
not sure what you mean here, this is what is already happening line 35? :thinking: